apps/system/js/task_manager.js

```diff
@@ -200,6 +203,10 @@
     this.element.addEventListener('touchstart', this);
     this.element.addEventListener('touchmove', this);
     this.element.addEventListener('touchend', this);
+    this.element.addEventListener('transitionend', function cs_transition() {
```

Comment:
Because of the bind() the listener and cs_transition are not the same thing, so removeEventListener wont have any effect here. You'll either need do this in handleEvent or use a var self = this closure.